### PR TITLE
simple-udp: added support for customizing labels and annotations by the caller

### PR DESCRIPTION
**Pull request by jkowalski**
ANNOTATION <key> <value>
LABEL <key> <value>

are now supported in addition to no-parameters option

### CODE REVIEW

**Comment by jkowalski** [comment]
should I split this one in two? first PR to make new release but don't use i yet, second to update all references 0.5=>0.6 ?

**Comment by markmandel** [comment]
I'll grab the PR and push up the new image for you - just been busy with cleanup and stuff. It's on my todo list (and then I'll rebuild the PR once it's done)

File: site/content/en/blog/releases/0.7.0.md [hunk 1]
```diff
@@ -44,7 +44,7 @@ Images available with this release:
 - [gcr.io/agones-images/agones-controller:0.7.0](https://gcr.io/agones-images/agones-controller:0.7.0)
 - [gcr.io/agones-images/agones-sdk:0.7.0](https://gcr.io/agones-images/agones-sdk:0.7.0)
 - [gcr.io/agones-images/cpp-simple-server:0.4](https://gcr.io/agones-images/cpp-simple-server:0.4)
-- [gcr.io/agones-images/udp-server:0.5](https://gcr.io/agones-images/udp-server:0.5)
+- [gcr.io/agones-images/udp-server:0.6](https://gcr.io/agones-images/udp-server:0.6)
```
Comment (markmandel) [inline site/content/en/blog/releases/0.7.0.md]: ooh, we should get rid of this - this is an old blog post

Comment (jkowalski) [reply site/content/en/blog/releases/0.7.0.md]: fixed that.

**Pull request merged**